TutorLizard.BusinessLogic/Data/RepositoryBase.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.Json;
+using TutorLizard.BusinessLogic.Models;
+using System.Diagnostics;
+
+namespace TutorLizard.BusinessLogic.Data
+{
+    public abstract class RepositoryBase<T>
+    {
+        protected List<T> _data = new();
+
+        private string _filePath = @"Data/users.json";
+
+        public RepositoryBase(string filePath)
+        {
+            _filePath = filePath;
+            LoadFromJson();
+        }
+
+
+        protected void LoadFromJson()
+        {
+            string filePath = Path.Combine(_filePath.Split('/'));
+            string fullPath;
+            if (Path.IsPathRooted(filePath))
+                fullPath = filePath;
+            else
+                fullPath = Path.Combine(AppContext.BaseDirectory, filePath);
+
+            if (!File.Exists(fullPath))
+                _data = new List<T>();
```

Comment:
Replace this line with `return;`
We changed return to assignment, but it's actually important to return here (but the value is not important).
`_data` initializes itself, so there's no need to assign empty list to it.

Reply:
This forced me to make few more changes (so the code looks like in DA now)

Reply:
If you did `return;` instead of `return new List<T>();` you wouldn't have to make these changes.